src/sr/templates.js

```diff
@@ -1553,7 +1553,7 @@ class AlgorithmIdentification extends Template {
 
 class TrackingIdentifier extends Template {
     constructor(options) {
-        super(options);
+        super();
```

Comment:
It isn't just this one, but every Template extension (5 of them) in this appears to be wrong.  Can you go through them and update the others?